domain/src/DomainValidator.php

```diff
@@ -104,7 +105,7 @@ public function validate(DomainInterface $domain) {
       }
     }
     // Allow modules to alter this behavior.
-    \Drupal::moduleHandler()->invokeAll('domain_validate', $error_list, $hostname);
+    \Drupal::moduleHandler()->invokeAll('domain_validate', array($error_list, $hostname));
```

Comment:
Do we have a test for this yet? 

I will admit that I found the new hook syntax confusing.